src/main/scala/no/ndla/auth/AuthProperties.scala

```diff
@@ -33,7 +33,10 @@ object AuthProperties extends LazyLogging {
 
   val Environment = propOrElse("NDLA_ENVIRONMENT", "local")
 
-  lazy val Domain = Domains.get(Environment)
+  lazy val Domain = Map(
```

Comment:
Merk at det som legges i `Domains` i `network` må gjelde for alle komponenter (eller i hvert fall de fleste). Betyr det at alle sammen skal bruke `proxy.ndla-local` som host for all lokal-kommunikasjon?